source/api/commands/type.md

```diff
@@ -53,9 +53,16 @@ Sequence | Notes
 `{pagedown}` | Scrolls down
 `{pageup}` | Scrolls up
 `{rightarrow}` | Moves cursor right
-`{selectall}` | Selects all text by creating a `selection range`
 `{uparrow}` | Moves cursor up
 
+Text passed to `.type()` may include special actions to be performed during typing:
```

Comment:
these are still `special character sequences` and they will be affected by `parseSpecialCharSequences` so it seems strange to call them `special actions` here. Maybe clarify that these are special char seqs that perform actions instead of single keys?

Reply:
yeah, it was a bit different so that's why I split it, but yah, I can see how this is important to clarify since the option still applies. Will update.